djstripe/models.py

```diff
@@ -106,7 +124,7 @@ def __str__(self):
 @python_2_unicode_compatible
 class Event(StripeObject):
     stripe_api_name = "Event"
-
+    objects = models.Manager()
```

Comment:
Is there an issue with just letting each object use the ``StripeObjectManager``?